ui/job-view/details/BugFiler.jsx

```diff
@@ -367,15 +369,16 @@ export class BugFilerClass extends React.Component {
       crashSignatures,
     } = this.state;
     const { toggle, successCallback, notify } = this.props;
-    const [product, component] = selectedProduct.split(' :: ');
 
     if (!selectedProduct) {
       notify(
         'Please select (or search and select) a product/component pair to continue',
         'danger',
       );
       return;
+
     }
+    const [product, component] = selectedProduct.split(' :: ');
```

Comment:
Move this below the notify() because if product or component are undefined, the notify() would never have been called do to a silent JS error.